src/iidm/BusBreakerVoltageLevel.cpp

```diff
@@ -359,6 +359,65 @@ void BusBreakerVoltageLevel::removeTopology() {
     removeAllBuses();
 }
 
+void BusBreakerVoltageLevel::traverse(BusTerminal& terminal, VoltageLevel::TopologyTraverser& traverser) const {
+    std::vector<std::reference_wrapper<Terminal>> traversedTerminals;
+    traverse(terminal, traverser, traversedTerminals);
+}
+
+void BusBreakerVoltageLevel::traverse(BusTerminal& terminal, VoltageLevel::TopologyTraverser& traverser, std::vector<std::reference_wrapper<Terminal>>& traversedTerminals) const {
```

Comment:
In Java, we use a `Set` instead of a `vector`. Maybe this could have an impact on algorithms that use the `traverse` method.
It could also lead to a performance issue